IBAnimatable/AnimatableModalViewController.swift

```diff
@@ -91,62 +116,66 @@ open class AnimatableModalViewController: UIViewController, PresentationDesignab
       presenter?.presentationConfiguration?.blurOpacity = blurOpacity
     }
   }
-
+  
   @IBInspectable public var shadowColor: UIColor? {
     didSet {
       presenter?.presentationConfiguration?.shadowColor = shadowColor
     }
   }
-
+  
   @IBInspectable public var shadowRadius: CGFloat = 0.7 {
     didSet {
       presenter?.presentationConfiguration?.shadowRadius = shadowRadius
     }
   }
-
+  
   @IBInspectable public var shadowOpacity: CGFloat = CGFloat.nan {
     didSet {
       presenter?.presentationConfiguration?.shadowOpacity = shadowOpacity
     }
   }
-
+  
   @IBInspectable public var shadowOffset: CGPoint = .zero {
     didSet {
       presenter?.presentationConfiguration?.shadowOffset = shadowOffset
     }
   }
-
-  @IBInspectable public var keyboardTranslation: String? {
+  @IBInspectable var _keyboardTranslation: String = "" {
     didSet {
-      presenter?.presentationConfiguration?.keyboardTranslation = ModalKeyboardTranslation.fromString(string: keyboardTranslation)
+      keyboardTranslation = ModalKeyboardTranslation(string: _keyboardTranslation) ?? .none
     }
   }
-
+  public var keyboardTranslation: ModalKeyboardTranslation = .none {
+    didSet {
+      presenter?.presentationConfiguration?.keyboardTranslation = keyboardTranslation    }
+  }
+  
   // MARK: Private
-
+  
   fileprivate var presenter: PresentationPresenter?
-
+  
   // MARK: Life cycle
-
+  
   public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
     super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
     setupPresenter()
   }
-
+  
   public required init?(coder aDecoder: NSCoder) {
     super.init(coder: aDecoder)
     setupPresenter()
   }
-
+  
   // MARK: Life cycle
-
+  
```

Comment:
I checked, Swxftlint don't say anything here. Actually, it's because I made Xcode Auto re-indent ( Editor -> Structure -> Re-indent)